virt/node_network/virt-updating-node-network-config.adoc

```diff
@@ -36,6 +36,18 @@ include::modules/virt-example-ethernet-nncp.adoc[leveloffset=+2]
 
 include::modules/virt-example-nmstate-multiple-interfaces.adoc[leveloffset=+2]
 
+[id="capturing-nic-static-ip_virt-updating-node-network-config"]
+== Capturing static IP of a NIC attached to a bridge
+
+ifdef::openshift-enterprise[]
+:FeatureName: Capturing static IP of a NIC
```

Comment:
Should there be a "the" before "static IP"? This would also apply to the title of this section.